### DISCUSSION TITLE
连接组件的链接可以带参数吗？

### DISCUSSION THREAD

**Discussion by jin1031960096** [jd-opensource/drip-table#19] [Q&A]
连接组件的链接可以带参数吗？
![C3835515-6483-447D-9799-50D5AB879645](https://user-images.githubusercontent.com/61078465/146749145-60b2ffee-01fc-4432-b604-5cbd0f9533c6.jpeg)

**Comment by helloqian12138** [comment]
目前链接组件还无法解析路由参数或者query参数，建议提个issue，然后我们会优先处理。
